### ISSUE TITLE
Replace Deprecated Test Case Assertion Methods in twisted.internet

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6221]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)| @Julian reported|
|-|-|
|Trac ID|trac#6221|
|Type|enhancement|
|Created|2012-12-26 03:26:12Z|
|Branch|https://github.com/twisted/twisted/tree/remove-deprecated-test-methods-internet-6221|

See [#5771](https://github.com/twisted/twisted/issues/5771)

Attachments:

* [internet-deprecate.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/df6/df6d2950dd36019d5d6e01f38366f76f5c3012ae/50f49a5860fb58dc174ea2242dc20b20b52acd27.patch) (3234 bytes) - added by Julian on 2012-12-26 03:26:20Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6221 6221
type__enhancement enhancement
reporter__Julian Julian
priority__normal normal
milestone__ 
branch__branches_remove_deprecated_test_methods_internet_6221 branches/remove-deprecated-test-methods-internet-6221
branch_author__julian julian
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1356492372000000 1356492372000000
changetime__1372873291000000 1372873291000000
version__None None
owner__Julian Julian

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_1"></a>|@tomprince commented|
|-|-|

There are a couple of methods to be deprecated that haven't been changed. At least ```assertIdentical``` and ```assertNotEquals```.

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)|@tomprince set owner to @Julian|
|-|-|

**Comment by twisted-trac** [comment]
|<img alt="julian's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|julian commented|
|-|-|

(In [38991]) Branching to remove-deprecated-test-methods-internet-6221.

**Comment by twisted-trac** [comment]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)<a name="note_4"></a>|@Julian removed owner|
|-|-|

Up for review.

See list of substitutions in [#6220](https://github.com/twisted/twisted/issues/6220) if you'd like the list of what should be being replaced.

[Build results](http://buildbot.twistedmatrix.com/boxes-supported?branch=/branches/remove-deprecated-test-methods-internet-6221)

**Comment by twisted-trac** [comment]
|[<img alt="tomprince's avatar" src="https://avatars.githubusercontent.com/u/283816?s=50" width="50" height="50">](https://github.com/tomprince)<a name="note_5"></a>|@tomprince set owner to @Julian|
|-|-|

This looks good, thanks. Please merge after adding a news file.

**Comment by twisted-trac** [comment]
|<img alt="julian's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|julian set status to `closed`|
|-|-|

(In [39017]) Merge remove-deprecated-test-methods-internet-6221: Replace to-be-deprecated assertions in twisted.internet

Author: Julian
Reviewers: tom.prince
Fixes: [#6221](https://github.com/twisted/twisted/issues/6221)

Replace older test method assertions like fail* (which are soon to be
deprecated) in twisted.internet's test suite with their newer equivalents.

**Comment by twisted-trac** [comment]
|[<img alt="Julian's avatar" src="https://avatars.githubusercontent.com/u/329822?s=50" width="50" height="50">](https://github.com/Julian)<a name="note_7"></a>|@Julian commented|
|-|-|

Thanks Tom!
